### PR TITLE
Nullrods now blocks vamp spells, unless vamp is young

### PR DESCRIPTION
**Pull request by ShiftyRail**
[balance]
Technically a balance thing.

_Why_ : because it gives people a way around shitty vamps ; your spell is not used if it can be blocked, so you don't waste your cooldown. (can be changed if needed).

:cl:
- tweak: To bring vampire counters more in-line with each other, the chaplain is now immune to spells from mature vampires instead of from young vampires. His holy weapon also now provides the same protection to whoever carries it.

### CODE REVIEW

**Comment by Irradiation** [comment]
I'm going to say go the other way around, because that's how I pseudo rebalanced vampire. IE: Stronk vampires get buttfucked more by holy stuff than lesser ones, etc. You let the early vamp get his welfare succs, then you can engage in true holy war against a worthy foe (Read: fucking annoying to fight). Your current way of looking at it is making early vamps gets buttfucked, then letting the mature ones go rampant.

TL;DR be consistent about how we treat vampires in regards to holy things.

:man_shrugging:

**Comment by ShiftyRail** [comment]
Did that, made a CL

**Comment by Intigracy** [comment]
I feel that bringing vamp counters to be consistent with other vamp counters is a good thing. They're already blocked by holy shit, the null rod is holy shit.

Maybe we can salvage it to be a less boring antag.
